runtime/src/bank.rs

```diff
@@ -1323,6 +1323,8 @@ impl Bank {
 
     pub fn deposit(&self, pubkey: &Pubkey, lamports: u64) {
         let mut account = self.get_account(pubkey).unwrap_or_default();
+        self.collected_rent
```

Comment:
could you capture this in store()?

Reply:
Nah, can't capture it in `store_account`, as it is used during processing genesis, adding instruction processor and updating sysvar too.

Reply:
I don't understand this statement.  sysvars are rent-exempt, by special case.  assessing rent via store_account() at genesis time would be fine.

Reply:
@rob-solana sysvars are not rent-exempt at the moment. But, I think that could be easily done. 
I understand now, If we capture this in `store_account()`, we can be sure, that there isn't a path left, where account is stored, without deducting rent.

Reply:
hoping so, yes ;)

Reply:
@rob-solana See PRs: https://github.com/solana-labs/solana/pull/7015 and https://github.com/solana-labs/solana/pull/7017

Reply:
@rob-solana Issue opened: https://github.com/solana-labs/solana/issues/7056